gpii/node_modules/gpii-oauth2/gpii-oauth2-datastore/src/DbDataStoreUtils.js

```diff
@@ -17,24 +17,12 @@ var fluid = fluid || require("infusion"),
     $ = fluid.registerNamespace("jQuery"),
     uuid = uuid || require("node-uuid");
 
-if (!gpii.oauth2.errors) {
+if (!gpii.oauth2.parseBearerAuthorizationHeader) {
```

Comment:
This is a bit of an odd test - is the intention to test whether we are on the client or not? If so, we could use fluid.contextAware.isBrowser() or simply if (typeof(require) !== "undefined")